functionary/core/models/parameter.py

```diff
@@ -20,7 +13,43 @@
 )
 
 
-class WorkflowParameter(models.Model):
+class Parameter(models.Model):
+    """Base model for common components of FunctionParameter and WorkflowParameter"""
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
Need to update the urls.py, it's expecting the parameter id to be an int for workflow parameter edit and delete

Reply:
Good catch.  Fixed.